federationapi/routing/state.go

```diff
@@ -107,7 +107,7 @@ func getState(
 		return nil, &util.JSONResponse{Code: http.StatusNotFound, JSON: nil}
 	}
 
-	prevEventIDs := getIDsFromEventRef(event.PrevEvents())
+	prevEventIDs := []string{eventID}
```

Comment:
Why aren't we using `event.PrevEvents()` anymore? Variable name indicates that's the correct behaviour...

Reply:
I'll need to rename the variable but it looks like using `event.PrevEvents()` was actually incorrect behaviour, [the spec](https://matrix.org/docs/spec/server_server/r0.1.3#get-matrix-federation-v1-state-roomid) states it should return "at" a given event.

Reply:
I've moved that inline a couple of lines down so the variable name is no longer confusing.